google-api-grpc/google-cloud-resource-name-common/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-resource-name-common</artifactId>
+  <version>0.12.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-resource-name-common:current} -->
+  <name>proto-google-cloud-websecurityscanner-v1alpha</name>
+  <description>PROTO library for proto-google-cloud-websecurityscanner-v1alpha</description>
```

Comment:
Remove the "PROTO" word from the description

Reply:
Done